targets/curie_bsp/setup.py

```diff
@@ -0,0 +1,221 @@
+                                project_name)
+    if not os.path.exists(location_path):
+        os.makedirs(location_path)
+    os.chdir(location_path)
+    # arc
+    if not os.path.islink(os.path.join(location_path, 'arc')):
+        os.symlink(os.path.join(jerry_path,
+                    'targets/curie_bsp/jerry_app/arc'), 'arc')
+
+    # include
+    if not os.path.islink(os.path.join(location_path, 'include')):
+        os.symlink(os.path.join(jerry_path,
+                    'targets/curie_bsp/jerry_app/include'), 'include')
+    # quark
+    if not os.path.islink(os.path.join(location_path, 'quark')):
+        os.symlink(os.path.join(jerry_path,
+                    'targets/curie_bsp/jerry_app/quark'), 'quark')
+
+    # jerryscript
+    location_path = os.path.join(location_path, 'quark')
+    os.chdir(location_path)
+    if not os.path.islink(os.path.join(location_path, 'jerryscript')):
+        os.symlink(jerry_path, 'jerryscript')
+
+
+# create Kbuild.mk
+def breadth_first_travel(root_dir):
+    out_put = ''
+    lists = os.listdir(root_dir)
+    dirs = []
+    for line in lists:
+        full_path = os.path.join(root_dir, line)
+        if os.path.isdir(full_path):
+            dirs.append(line)
+        else:
+            c_file = line.endswith('.c')
+            if c_file:
+                npos = line.find('.c')
+                out_put += 'obj-y += ' + line[0:npos] + '.o\n'
+                continue
+            asm_file = line.endswith('.S')
+            if asm_file:
+                npos = line.find('.S')
+                out_put += 'obj-y += ' + line[0:npos] + '.o\n'
+    for line in dirs:
+        out_put += 'obj-y += ' + line + '/\n'
+
+    file_path = os.path.join(root_dir, 'Kbuild.mk')
+    file_to_be_created = open(file_path, 'w+')
+    file_to_be_created.write(out_put)
+    file_to_be_created.close()
+    for line in dirs:
+        breadth_first_travel(os.path.join(root_dir, line))
+
+
+# create Kbuild.mk in jerryscript/
+def create_kbuild_in_jerryscript(jerry_path):
+    breadth_first_travel(os.path.join(jerry_path, 'jerry-core'))
+    breadth_first_travel(os.path.join(jerry_path, 'jerry-libm'))
+    #jerryscript/
+    out_put = '''
+obj-y += jerry-core/
+obj-y += jerry-libm/
+obj-y += targets/
+subdir-cflags-y += -DCONFIG_MEM_HEAP_AREA_SIZE=10*1024
+subdir-cflags-y += -DJERRY_NDEBUG
+subdir-cflags-y += -DJERRY_DISABLE_HEAVY_DEBUG
+subdir-cflags-y += -DCONFIG_DISABLE_NUMBER_BUILTIN
+subdir-cflags-y += -DCONFIG_DISABLE_STRING_BUILTIN
+subdir-cflags-y += -DCONFIG_DISABLE_BOOLEAN_BUILTIN
+subdir-cflags-y += -DCONFIG_DISABLE_ERROR_BUILTINS
+subdir-cflags-y += -DCONFIG_DISABLE_ARRAY_BUILTIN
+subdir-cflags-y += -DCONFIG_DISABLE_MATH_BUILTIN
+subdir-cflags-y += -DCONFIG_DISABLE_JSON_BUILTIN
+subdir-cflags-y += -DCONFIG_DISABLE_DATE_BUILTIN
+subdir-cflags-y += -DCONFIG_DISABLE_REGEXP_BUILTIN
+subdir-cflags-y += -DCONFIG_DISABLE_ANNEXB_BUILTIN
+#subdir-cflags-y += -DCONFIG_ECMA_COMPACT_PROFILE_DISABLE_ERROR_BUILTIN
```

Comment:
Eventhough commented out, this line still refers to a macro that does not exist anymore